# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1251188761 -7200
#      Tue Aug 25 10:26:01 2009 +0200
# Branch release-3-2-x
# Node ID 4adc2a0690310471b7c8291670c2d8459dd4b639
# Parent  d2cd38f7608e226e8ba834dee57e29fa54366199
imread.m: undo unintended change

diff --git a/scripts/image/imread.m b/scripts/image/imread.m
--- a/scripts/image/imread.m
+++ b/scripts/image/imread.m
@@ -49,17 +49,17 @@ function varargout = imread (filename, v
 
   fn = file_in_path (IMAGE_PATH, filename);
 
   if (isempty (fn))
     error ("imread: cannot find %s", filename);
   endif
 
   try
-    [varargout{1:nargout}] = magick_read_internal (fn, varargin{:});
+    [varargout{1:nargout}] = __magick_read__ (fn, varargin{:});
   catch
 
     magick_error = lasterr ();
 
     img_field = false;
     x_field = false;
     map_field = false;
 
